data-transform.qmd

```diff
@@ -71,8 +71,8 @@ But before we discuss their individual differences, it's worth stating what they
 
 3.  The result is always a new data frame.
 
-Because the first argument is a data frame and the output is a data frame, dplyr verbs work well with the pipe, `|>`.
-The pipe takes the thing on its left and passes it along to the function on its right so that `x |> f(y)` is equivalent to `f(x, y)`, and `x |> f(y) |> g(z)` is equivalent to into `g(f(x, y), z)`.
+Since each verb is quite simple, solving complex problems will usually require combining multiple verbs, and we'll do so with the pipe, `|>`.
+We'll come back to discuss the pipe more in @the-pipe, but in brief, the pipe takes the thing on its left and passes it along to the function on its right so that `x |> f(y)` is equivalent to `f(x, y)`, and `x |> f(y) |> g(z)` is equivalent to into `g(f(x, y), z)`.
```

Comment:
Just above we said "we'll come back", to say again "we'll come back" sounds a bit odd to me.